README.md

```diff
@@ -1,2 +1,179 @@
-# stellar-docs
-Documentation for Stellar
+# Stellar Documentation and API Reference
+
+Welcome to the official home repository for [Documentation][docs] and
+[API Reference][api] for the [Stellar Network][stellar].
+
+# Table of Contents
+
+- [Stellar Documentation and API Reference](#stellar-documentation-and-api-reference)
+- [Table of Contents](#table-of-contents)
+- [Contributing](#contributing)
+- [Quick Start](#quick-start)
+- [Repository Structure](#repository-structure)
+- [Using Markdown](#using-markdown)
+  - [Markdown Basics](#markdown-basics)
+  - [Custom Markdown](#custom-markdown)
+    - [Alert](#alert)
+    - [Code Example](#code-example)
+
+# Contributing
+
+Contributions are more than welcome! Thank you! 🎉
+
+Before diving in, please read our [Stellar Contribution Guide][contrib] for
+details on contributing to Stellar's various repositories. Please take special
+note of the code of conduct and contributor license agreement.
+
+Our documentation site is built using [Docusaurus][docusaurus]. The content is
+written in [MDX][mdx], which adds a lot of cool possibilities. Even if you're
+unfamiliar with plain markdown, do not fear! You can still contribute in a
+helpful and meaningful way.
```

Comment:
Do we explain somewhere in here how you can contribute if you don't know markdown? You may have and I just missed it.

Reply:
No, I don't think we do lol

I mean, people can do this kind of review thing, help with the docusaurus stuff, fix typos, etc.